src/main/java/org/jnode/fs/xfs/XfsEntry.java

```diff
@@ -116,16 +114,6 @@ public long getInodeLastChanged() {
         return getMilliseconds(inode.getInodeChangeTimeSec(), inode.getInodeChangeTimeNsec());
     }
 
-    @Override
-    public long getCreated() {
```

Comment:
Move this one into InodeV3, the reason is if we add it here, all nodes other than v3 will go into the else block here, which throws massive exception.